_topic_maps/_topic_map.yml

```diff
@@ -2191,6 +2191,9 @@ Topics:
 - Name: Performance Addon Operator for low latency nodes
   File: cnf-performance-addon-operator-for-low-latency-nodes
   Distros: openshift-origin,openshift-enterprise
+- Name: Topology Aware Lifecycle Operator for ZTP cluster updates
```

Comment:
Why do we need "for ZTP cluster updates"? TALO can be used for non ZTP clusters as well.

Reply:
Will update titles accordingly